consensus/src/epoch_manager.rs

```diff
@@ -417,6 +418,28 @@ impl EpochManager {
         spawn_named!("Quorum Store", quorum_store.start());
     }
 
+    fn spawn_block_retrieval_task(&mut self, epoch: u64, block_store: Arc<BlockStore>) {
+        let (request_tx, mut request_rx) = aptos_channel::new(
+            QueueStyle::LIFO,
+            1,
```

Comment:
Why allow only one message on this channel?

Reply:
it's for the DDoS prevention, we only care about the last block retrieval request from any peer